### PR TITLE
Remove yarn and better repo cleanup

### PR DESCRIPTION
**Pull request by robertsLando**
Starting from zui 9.5.0 yarn has been removed and directory structure has changed a bit.

### CODE REVIEW

**Comment by robertsLando** [comment]
Hi @frenck, is there something missing here?

**Comment by frenck** [comment]
CI is failing?

File: zwave-js-ui/Dockerfile [hunk 1]
```diff
@@ -26,41 +26,32 @@ RUN \
...
-    && npm uninstall -g yarn \
     && apk del --no-cache --purge .build-dependencies \
+    && find ./opt -mindepth 1 -maxdepth 1 \
+    ! -name "node_modules" \
+    ! -name "snippets" \
+    ! -name ".git" \
+    ! -name "package.json" \
+    ! -name "server" \
+    ! -name "dist" \
+    -exec rm -rf {} +
```
Comment (frenck) [inline zwave-js-ui/Dockerfile:52]:
```suggestion
    -exec rm -rf {} + \
```

File: zwave-js-ui/Dockerfile [hunk 2]
```diff
@@ -26,41 +26,32 @@ RUN \
...
+        npm rebuild @serialport/bindings-cpp \
+    && npm prune --omit=dev
+    && npm cache clean \
     \
-    && npm uninstall -g yarn \
     && apk del --no-cache --purge .build-dependencies \
+    && find ./opt -mindepth 1 -maxdepth 1 \
+    ! -name "node_modules" \
+    ! -name "snippets" \
+    ! -name ".git" \
```
Comment (frenck) [inline zwave-js-ui/Dockerfile:48]: Not sure why git matters?

Comment (robertsLando) [reply zwave-js-ui/Dockerfile:48]: It can be ignored but I usually keep it so I can get the sha in the version number

Comment (frenck) [reply zwave-js-ui/Dockerfile:48]: It cleans up an non-existing folder. The directory doesn't exist in the release...

File: zwave-js-ui/Dockerfile (hunk 1 above)
Comment (frenck) [reply zwave-js-ui/Dockerfile:52]: In that case, please don't open PRs. Drive-by's are not useful, often contains errors that slip in or cause unneeded review overhead.

Instead file an issue, or better, test the PR before creating.

File: zwave-js-ui/Dockerfile (hunk 2 above)
Comment (robertsLando) [reply zwave-js-ui/Dockerfile:48]: Yeah right you are not cloning the project, you can ignore it so :)

**Pull request closed without merging**